agenta-backend/agenta_backend/ee

```diff
@@ -1 +0,0 @@
-../ee_backend/ee
```

Comment:
This is the symbolic link to the ee folder. Removing it breaks cloud to my knowledge.

Reply:
@mmabrouk -- would you be able to add it back ?